relay-server/src/endpoints/minidump.rs

```diff
@@ -31,6 +38,12 @@ const MINIDUMP_FILE_NAME: &str = "Minidump";
 const MINIDUMP_MAGIC_HEADER_LE: &[u8] = b"MDMP";
 const MINIDUMP_MAGIC_HEADER_BE: &[u8] = b"PMDM";
 
+/// Magic headers of the currently supported minidump compression containers.
+const GZIP_MAGIC_HEADER: &[u8] = b"\x1F\x8B";
+const XZ_MAGIC_HEADER: &[u8] = b"\xFD\x37\x7A\x58\x5A\x00";
+const BZIP2_MAGIC_HEADER: &[u8] = b"\x42\x5A\x68";
+const ZSTD_MAGIC_HEADER: &[u8] = b"\x28\xB5\x2F\xFD";
```

Comment:
While a bit redundant, we should document all constants individually. If you query for docs with rust-analyzer then it'll show up.

Rephrased a bit, but not sure if that makes it better?
```suggestion
/// Magic bytes for gzip compressed minidump containers.
const GZIP_MAGIC_HEADER: &[u8] = b"\x1F\x8B";
/// Magic bytes for gzip compressed minidump containers.
const XZ_MAGIC_HEADER: &[u8] = b"\xFD\x37\x7A\x58\x5A\x00";
/// Magic bytes for gzip compressed minidump containers.
const BZIP2_MAGIC_HEADER: &[u8] = b"\x42\x5A\x68";
/// Magic bytes for zstd compressed minidump containers.
const ZSTD_MAGIC_HEADER: &[u8] = b"\x28\xB5\x2F\xFD";
```